src/App.js

```diff
@@ -5,6 +5,8 @@ import Landing from "./Pages/Landing";
 import Login from "./Pages/Login";
 import Signup from "./Pages/Signup";
 
+import { RequiresAuth } from "./Components/RequiresAuth/RequiresAuth";
```

Comment:
You can have index.js file in your Components folder and export all files from here instead of importing them individually you can directly import from index.js of Components folder.

Reply:
I've done that for individual components but for full pages I have imported them like this